### DISCUSSION TITLE
Disable music video downloads?

### DISCUSSION THREAD

**Discussion by ss1gohan13** [RandomNinjaAtk/docker-lidarr-extended#118] [Q&A] [answered]
Hello,

I just noticed that Lidarr-extended is now downloading music videos by default. Is there an option to disable this feature?

Thanks,

**Comment by RandomNinjaAtk** [comment, marked as answer]
Read the readme?
Link: https://github.com/RandomNinjaAtk/docker-lidarr-extended#parameters

You can set one of the parameters to enable/disable the feature.
